### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Added image folder of the webpage.
Added HTML File of the project.
Added Css File of the project.
Added MIT File of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+internet
+ Setup
+
+-Install
+
+-Usage
+Micronuate
+
+-Deployment
+internet
```
Comment (AnselemOdims) [inline README.md:33]: - [ ] Kindly fill up these empty sections or remove them completely. This will make the README more professional.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Godfrey-Alimony , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] There should be a `.gitignore` file, that includes `node_modules` and `test.md`. Kindly make sure you include it as it will help ensure that unnecessary files or dependencies are not pushed to remote.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Godfrey-Alimony** [comment]
Added README File of the Project.
Added comment on the HTML and CSS File.
Added .gitignore file and node_module,test.md Under it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Godfrey-Alimony , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [ ]  Kindly create a .gitignore file, that includes node_modules and test.md So that unnecessary dependencies are not pushed to the remote. Kindly implement said by the previous reviewer.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Godfrey-Alimony** [comment]
Dear sir/madam,
How are you? please check i have created gitignore file which include
node_modules and test.md.
Thanks

**Comment by roshan-bajgain** [comment]
@Godfrey-Alimony If you are done then I suggest you submit the same pull request on your dashboard.

**Comment by Godfrey-Alimony** [comment]
Yes, I have submitted it.
Thanks
[image: Screenshot from 2022-04-20 09-45-09.png]

**Comment by Godfrey-Alimony** [comment]
https://github.com/Godfrey-Alimony/Hello-World/pull/1

**Comment by Godfrey-Alimony** [comment]
Thank you so much sir.

**Comment by Godfrey-Alimony** [comment]
Dear sir;
How are you? well i have done all that I could please approve my project so
that  I start the next one.Thanks

**Pull request merged**